lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -148,6 +149,15 @@ def symlink_plugin_paths(source_dir, target_path)
           end
         end
       end
+
+      def cleanse_locale_files
```

Comment:
Since this is not a straight delete, but actually a proper step in the build process, let's rename this method... Something like `def compile_locales`